### PR TITLE
Simulation Test

### PR DESCRIPTION
**Pull request by michaeleh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,12 +12,11 @@
...
     package_dir={"": "src"},
     packages=["corona_hakab_model", "corona_hakab_model_data"],
-    install_requires=["numpy", "scipy", "scikit-learn", "matplotlib"],
+    install_requires=["numpy", "scipy", "scikit-learn","matplotlib", "pytest"],
     extras_require={
         "quality": ["pytest", "tox", "flake8", "isort", "black"],
         "profiling": ["yappi"],
-        "pretty graphs": ["pyside2"],
-        "building": ["swimport"],
+        "pretty graphs": ["pyside2", "matplotlib"],
```
Comment (meirhalachmi) [inline setup.py:19]: matplotlib is already required above, we don't need it as an extra too

File: setup.py [hunk 2]
```diff
@@ -12,12 +12,11 @@
     description="Corona Hakab Model Simulator",
     package_dir={"": "src"},
     packages=["corona_hakab_model", "corona_hakab_model_data"],
-    install_requires=["numpy", "scipy", "scikit-learn", "matplotlib"],
+    install_requires=["numpy", "scipy", "scikit-learn","matplotlib", "pytest"],
     extras_require={
         "quality": ["pytest", "tox", "flake8", "isort", "black"],
         "profiling": ["yappi"],
-        "pretty graphs": ["pyside2"],
-        "building": ["swimport"],
```
Comment (meirhalachmi) [inline setup.py:20]: why did you delete swimport?

File: tests/simulation_sanity_test/simulation_test.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+f = open('simulation_test_ranges.json', 'r')
+range_per_day_dict = json.load(f)
+
+
+def test_no_policy_simulation():
+    """
+    run the simulation with no policy and fixed R0
+    """
+    warnings.warn(UserWarning("Ranges for each state was generate from real sim runs, might be flaky."))
+    consts = Consts(active_isolation=False, population_size=1_000, r0=2.4)
```
Comment (meirhalachmi) [inline tests/simulation_sanity_test/simulation_test.py:17]: There are many default consts that might change and mess with the results. 
Maybe you should export the consts to an external file and load it in the test

File: setup.py (hunk 1 above)
Comment (michaeleh) [reply setup.py]: done

File: setup.py (hunk 2 above)
Comment (michaeleh) [reply setup.py]: done

File: tests/simulation_sanity_test/simulation_test.py (hunk 3 above)
Comment (michaeleh) [reply tests/simulation_sanity_test/simulation_test.py]: done
